### DISCUSSION TITLE
Difference with Path Intellisense?

### DISCUSSION THREAD

**Discussion by ackvf** [mihai-vlc/path-autocomplete#136] [Q&A]
I see that the Path [Intellisense extension by Christian Kohler](https://github.com/ChristianKohler/PathIntellisense) is mentioned in the Credits section and somehow, over the years, I ended up with both having installed and so I was wondering 
> **What is the actual difference between the two and which one is superior**?

Upon first glance and a bit of testing in various string contexts, I couldn't find the difference. I couldn't not notice, though, the sheer amount of configuration options this extension offers, but i am not able to test all these, because I may not even understand them all.

**Comment by mihai-vlc** [comment]
You can try both extensions individually and see what works best for your project/environment.
There is no need to have both extensions installed at the same time since you may end up with duplicate suggestions.

The most common customization options used in path autocomplete are:
- path-autocomplete.pathMappings
- path-autocomplete.transformations


You can find examples on how to use them in the readme file.
They are useful if you need to customize the suggestions (change file names, update the starting directory).

**Reply by ackvf** [reply]
Hi, I appreciate your time, however this seems like an evasive answer. I obviously tried both separately, but I did not find any differences in base usage. This is why I asked if someone more knowledgeable knows from the top of their head what differences there are. What are the most striking features of path-autocomplete that are missing in the original extension?
